docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -101,61 +101,14 @@ if there is an unreachability problem Split Brain Resolver would make a decision
 ## Configuration Compatibility Checks
 
 During rolling updates the configuration from existing nodes should pass the Cluster configuration compatibility checks.
-For example, it is possible to migrate Cluster Sharding from Classic to Typed Actors in a rolling update using a two step approach
-as of Akka version `2.5.23`:
+For example, it is possible to migrate Cluster Sharding from Classic to Typed Actors in a rolling update using a two step approach:
 
 * Deploy with the new nodes set to `pekko.cluster.configuration-compatibility-check.enforce-on-join = off`
 and ensure all nodes are in this state
 * Deploy again and with the new nodes set to `pekko.cluster.configuration-compatibility-check.enforce-on-join = on`. 
   
 Full documentation about enforcing these checks on joining nodes and optionally adding custom checks can be found in  
-@ref:[Akka Cluster configuration compatibility checks](../typed/cluster.md#configuration-compatibility-check).
-
-## Rolling Updates and Migrating Akka
-
-### From Java serialization to Jackson
- 
-If you are migrating from Akka 2.5 to 2.6, and use Java serialization you can replace it with, for example, the new
```

Comment:
Every time a concrete Akka version was mentioned, I had to make a decision, because there is no Pekko equivalent. So here, I am removing this detailed migration guide